src/gpl/src/initialPlace.cpp

```diff
@@ -41,6 +43,40 @@
 #include "graphics.h"
 
 #include "utl/Logger.h"
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include "cusparse.h"
+#include "cusolverDn.h"
+#include "cusolverSp.h"
+
+#define CUDA_ERROR(value) do { \
+   cudaError_t m_cudaStat = value; \
+   if(m_cudaStat != cudaSuccess){ \
+       fprintf(stderr, "Error %s at line %d in file %s\n", cudaGetErrorString(m_cudaStat), __LINE__, __FILE__); \
+       exit(-1); \
+   } \
+} while(0)
+#define CUSPARSE_ERROR(value) do { \
+    cusparseStatus_t _m_status = value;\
```

Comment:
consistency: above, you use `m_xxx`, here you use `_m_xxx`. Picking one consistent naming scheme makes it easier for future readers of the code.